doc/21-development.md

```diff
@@ -1093,7 +1093,7 @@ Icinga application using a dist tarball (including notes for distributions):
   - SUSE: libopenssl-devel (for SLES 11: libopenssl1-devel)
   - Debian/Ubuntu: libssl-dev
   - Alpine: libressl-dev
-* Boost library and header files >= 1.48.0
+* Boost library and header files >= 1.53.0
   - RHEL/Fedora: boost148-devel
```

Comment:
Minor thing, the package names should be updated as well (one line below, I don't know if they proceed with that naming schema in Fedora. That's a good candidate for some research and a docs table.

Reply:
#ElephantOversight